velox/common/memory/MmapAllocator.cpp

```diff
@@ -544,6 +667,7 @@ MachinePageCount MmapAllocator::SizeClass::free(
       }
       if (bits::isBitSet(pageMapped_.data(), page)) {
         ++numMappedFreePages_;
+        markMappedFree(page);
```

Comment:
Why are we freeing the lookup bit when only one class page is freed in a range of class pages represented by a lookup bit?